ambry-router/src/main/java/com/github/ambry/router/UndeleteOperation.java

```diff
@@ -340,6 +362,13 @@ private void checkAndMaybeComplete() {
         operationException.set(
             new RouterException("UndeleteOperation failed because of BlobNotFound", RouterErrorCode.BlobDoesNotExist));
       }
+      if(quotaChargeCallback != null) {
```

Comment:
all of these code blocks seem to be missing a space in between if and paren.

Reply:
fixed.